app/services/add_receipt_to_c2_proposal_attributes.rb

```diff
@@ -0,0 +1,21 @@
+class AddReceiptToC2ProposalAttributes
```

Comment:
I feel like this class and the other attribute class belong in `app/models` more than `app/services` -- what do you think?